### ISSUE TITLE
Call for Consensus: publish Candidate Recommendation

### ISSUE THREAD

**Issue by ProgramMax** [w3c/PNG-spec#340]
This GitHub issue is to establish consensus on publishing a Candidate Recommendation. W3C Working Group participants should reply with affirmation, dissent, or may abstain.

Should we publish a Candidate Recommendation?

Process
[Consensus is a core value to the W3C.](https://www.w3.org/2021/Process-20211102/#Consensus) I have assigned this issue to all [PNG WG participants who have their GitHub profile listed](https://www.w3.org/groups/wg/png/participants). I was unable to include Simon Thompson in the assignees list. Pinging @simontWork . Also, Said Abou-Hallawa does not have a GitHub account listed. I will try to email them.

A WG participant may register a [Formal Objection](https://www.w3.org/2021/Process-20211102/#FormalObjection) if they would like Director consideration.

WG participants may respond with a simple yes/no or similar. They may also elaborate. If dissenting, elaboration is encouraged in order to identify issues and find solutions. The [W3C process on managing dissent](https://www.w3.org/2021/Process-20211102/#managing-dissent) states that we will try to address concerns as much as is reasonable and should move on once that is done.

Call to action
Working group participants should reply with their stance on publishing a Candidate Recommendation.

**Comment by lrosenthol** [comment]
Yes!

**Comment by shallawa** [comment]
Yes.

**Comment by chrisn** [comment]
Yes

**Comment by simontWork** [comment]
Apologies for the delay in replying, I've been on vacation.

I've read the document once more and I would prefer it if issues #341 #342 and #343 were addressed prior to CR.

**Comment by svgeesus** [comment]
> I would prefer it if issues https://github.com/w3c/PNG-spec/issues/341 https://github.com/w3c/PNG-spec/issues/342 and https://github.com/w3c/PNG-spec/issues/343 were addressed prior to CR.

I agree, and prefer these be addressed which seems easily possible. Once that is done, yes to requesting advancement to CR.

**Comment by svgeesus** [comment]
_(I am also on vacation)_

**Comment by svgeesus** [comment]
> I've read the document once more and I would prefer it if issues https://github.com/w3c/PNG-spec/issues/341 https://github.com/w3c/PNG-spec/issues/342 and https://github.com/w3c/PNG-spec/issues/343 were addressed prior to CR.

Would be fixed by https://github.com/w3c/PNG-spec/pull/345#pullrequestreview-1578910728

**Comment by svgeesus** [comment]
We now have [7 open issues tagged **blocking-3rd-edition-cr**](https://github.com/w3c/PNG-spec/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Ablocking-3rd-edition-cr).

Of those: https://github.com/w3c/PNG-spec/issues/337 has an open PR which would resolve it.

https://github.com/w3c/PNG-spec/issues/343 has good discussion on definitions and terms which would resolve it, so a PR would be easy to create.

https://github.com/w3c/PNG-spec/issues/346 has some early discussion, probably needs more

There is one Privacy and one Security issue, in both of those the discussion seems to have run its course, some edits have been made, and it is uclear what more needs to be done.

https://github.com/w3c/PNG-spec/issues/308 and https://github.com/w3c/PNG-spec/issues/194 are minor polish issues, nice to have, I really don't see them as blocking CR though.

Lastly, there is a publishing moratorium starting 8 September and approval of a request for Candidate Recommendation typically takes at least a week; so I don't see us publishing this until after TPAC. But we can at least make the request, which I am about to do, and clear up what issues we can between now and the end of TPAC.

**Comment by svgeesus** [comment]
@ProgramMax could you please declare, on this issue, whether we have consensus to request transition to Candidate Recommendation? Also your own +1 or -1 should be added.

So far we have 4 in favor, 5 once #343 is closed.

**Comment by ProgramMax** [comment]
+1 for me.
Given that all voices are either favorable or pending an issue that is making progress, I am okay saying we have consensus and transitioning to Candidate Recommendation.

**Comment by svgeesus** [comment]
[Transition to CR requested 31 Aug 2023](https://github.com/w3c/transitions/issues/566)

**Comment by svgeesus** [comment]
[Published 21 Sept 2023](https://www.w3.org/TR/2023/CR-png-3-20230921/)
